src/setup/styleguide.polyfills.ts

```diff
@@ -1,12 +1,10 @@
 /* eslint-disable prefer-rest-params, no-plusplus */
 /**
  * Node.replaceWith() polyfill.
- *
- * @param {Element} Ele - The new element.
- *
- * @constructor
  */
-function ReplaceWith(Ele) { // eslint-disable-line no-inner-declarations
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function ReplaceWith(Ele: any) { // eslint-disable-line no-inner-declarations
+  // @ts-ignore
```

Comment:
Can't the context for `this` somehow be defined? It's one of the 3 at the bottom of the file.

Reply:
Polyfill deleted as it was needed for IE11